test/cypress/integration/user/dashboard_spec.js

```diff
@@ -1,25 +1,42 @@
 describe('Dashboard', () => {
-  const selfServiceUsers = require('../../../fixtures/config/self_service_user.json')
+  const userExternalId = '7d19aff33f8948deb97ed16b2912dcd3'
+  const gatewayAccountId = '666'
+  const serviceName = 'Test Service'
 
   beforeEach(() => {
-    cy.setCookie('session', Cypress.env('encryptedSessionCookie'))
-    cy.setCookie('gateway_account', Cypress.env('encryptedGatewayAccountCookie'))
+    cy.task('getCookies', {
+      user_external_id: userExternalId,
+      gateway_account_id: gatewayAccountId
+    }).then(cookies => {
+      cy.setCookie('session', cookies.encryptedSessionCookie)
+      cy.setCookie('gateway_account', cookies.encryptedGatewayAccountCookie)
+    })
```

Comment:
Setup the cookies per test, before we did it globally by setting environment variables in plugins/index.js, but this assumed we were only ever using the same user/gateway account which wasn't the case for a recent test added so this got a bit messy.